dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/DefaultEventService.java

```diff
@@ -198,6 +198,11 @@ public Events getEvents(EventOperationParams operationParams)
     return Events.of(eventList, pager);
   }
 
+  @Override
+  public Set<String> canOrderEventsByFields() {
```

Comment:
Would something like `getOrderableFields` be more descriptive?
At first glance, I thought that this method was returning a boolean

Reply:
It was indeed a predicate before 😅 I did want to keep 'Events' in the the method name as only getEvents can be ordered. Not sure if this is important though. What do you think?